pkg/operator/ceph/csi/betav1csidriver.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+NOTE: Once support of older releases are removed; delete the file to
```

Comment:
```suggestion
TODO: Once support of older releases are removed; delete the file to
```

Reply:
Can we be more specific, which releases of what? ceph-csi? kubernetes? Do we have an idea on when we are planning on dropping this?

Reply:
> Can we be more specific, which releases of what? ceph-csi? kubernetes?

That must be an overlook, thanks for catching! Will update it.

> Do we have an idea on when we are planning on dropping this?

Since the betav1 CSIDriver object is deprecated from k8s 1.19+ and unavailable in v1.22+; we might need to keep support for betav1 CSIDriver until rook supports those kubernetes versions.